package.json

```diff
@@ -18,6 +18,7 @@
   },
   "homepage": "https://github.com/TrandafilTatiana/KMarketer#readme",
   "devDependencies": {
+    "@parcel/transformer-sass": "^2.13.3",
```

Comment:
do you really need this package? 

Parcel seems to work with sass out of the box